contracts/interfaces/IModuleFactory.sol

```diff
@@ -115,4 +133,11 @@ interface IModuleFactory {
      */
     function changeTitle(string calldata _title) external;
 
+    function polymathRegistry() external returns (address polymathRegistryAddress);
```

Comment:
Where is this used? Quick search doesn't show anything.

Reply:
I used this in the earlier implementation but afterwards requirements get change. I keep this because it doesn't hurt anything and make it compliant with ModuleFactory contract.